packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -315,7 +315,10 @@ export const prepareStatusManager = (
       },
 
       /**
-       * Add a new transaction with OBSERVED status
+       * Add a new transaction with OBSERVED status.
+       *
+       * This message isn't currently being sent.
```

Comment:
really? it's not?

Reply:
Please convince me I'm wrong. I couldn't find any instances of it.

Reply:
Changed in https://github.com/Agoric/agoric-sdk/issues/10994

Let's remove the function since it's not used.

Reply:
done